samples/routing/flowcontrol/Core_6/Client/Program.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Configuration.AdvanceExtensibility;
+using NServiceBus.Logging;
+using NServiceBus.Routing;
+using NServiceBus.Support;
+
+class Program
+{
+
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.FlowControl.Client";
+        const string letters = "ABCDEFGHIJKLMNOPQRSTUVXYZ";
+        var random = new Random();
+
+        var endpointConfiguration = new EndpointConfiguration("Samples.FlowControl.Client");
+        endpointConfiguration.UseSerialization<JsonSerializer>();
+        endpointConfiguration.UsePersistence<InMemoryPersistence>();
+        endpointConfiguration.EnableInstallers();
+        endpointConfiguration.SendFailedMessagesTo("error");
+
+        #region Routing
+        endpointConfiguration.UnicastRouting().RouteToEndpoint(typeof(PlaceOrder), "Samples.FlowControl.Server");
+        const string server = "Samples.FlowControl.Server";
+        endpointConfiguration.UnicastRouting().Mapping.Physical.Add(
+            new EndpointInstance(server, "1").AtMachine(RuntimeEnvironment.MachineName),
+            new EndpointInstance(server, "2").AtMachine(RuntimeEnvironment.MachineName));
+        #endregion
+
+        #region FairDistributionClient
+        endpointConfiguration.EnableFeature<FlowControl>();
+        endpointConfiguration.UnicastRouting().Mapping.SetMessageDistributionStrategy(
+            new ToLeastBusyDistributionStrategy(endpointConfiguration.GetSettings()), type => true);
```

Comment:
This is an old API. The new one operates on endpoints so after upgrading to latest core code it will turn into `SetMessageDistributionStrategy(myStrategy, "myEndpoint");`